### DISCUSSION TITLE
How to access Fobi Form data with code

### DISCUSSION THREAD

**Discussion by trexmo** [barseghyanartur/django-fobi#293] [General]
If I have a Form that I can enter data and retrieve the entries ok with Fobi.  Now I want to access that data in my application.  How do I do that?  I see DRF is integrated but cannot find the documentation on how to do it and the Demo site DRF integration is basically not helpful.

Things I want to do:

1. Post new form data from code in other parts of the application
2. Retrieve data from the forms to be used as parameters for another process in the application.
3. Do reporting from the data collected in the forms.

**Comment by barseghyanartur** [comment]
It's a valid question. So far, there are not `drf` views to access the submitted form data. You could however use `fobi` interface for that. I'll add it to the road-map.

**Comment by barseghyanartur** [comment]
Correspondent issue: https://github.com/barseghyanartur/django-fobi/issues/296

**Comment by trexmo** [comment]
Excellent, thanks.
